rocAL/include/pipeline/tensor.h

```diff
@@ -315,7 +313,7 @@ class Tensor : public rocalTensor {
         if (buffer)
             _mem_handle = buffer;
         else {
-            THROW("Invalid buffer pointer passed")
+            WRN("Invalid buffer pointer passed")
```

Comment:
why is this changing to a Warning? Shouldn't an invalid buffer pointer throw an error and exit?

Reply:
This was changed to handle one of the missing component behaviour (skip)